### ISSUE TITLE
npm start gets dependency failure. Flexbox with rxjs

### ISSUE THREAD

**Issue by sakthiwd** [ngx-rocket/generator-ngx-rocket#376]
## I'm submitting a...

- [x ] Bug report  
- [ ] Feature request
- [ ] Documentation issue or request

## Current behavior

ERROR in node_modules/@angular/flex-layout/core/typings/observable-media/observable-media.d.ts(11,14): error TS2416: Property 'subscribe' in type 'ObservableMedia' is not assignable to the same property in base type 'Subscribable<MediaChange>'.
  Type '(next?: (value: MediaChange) => void, error?: (error: any) => void, complete?: () => void) => Sub...' is not assignable to type '{ (observer?: PartialObserver<MediaChange>): Unsubscribable; (next?: (value: MediaChange) => void...'.
    Types of parameters 'next' and 'observer' are incompatible.
      Type 'PartialObserver<MediaChange>' is not assignable to type '(value: MediaChange) => void'.
        Type 'NextObserver<MediaChange>' is not assignable to type '(value: MediaChange) => void'.
          Type 'NextObserver<MediaChange>' provides no match for the signature '(value: MediaChange): void'.

## Expected behavior

Success

## Minimal reproduction of the problem with instructions

Tried up the following instructions as specied in ngx-rocket.
This happened when trying to "npm start".
Among my search, there is some conflicts between the flexlayout and rxjs. I have 6.30 version of rxjs right now

## Environment

```
- generator version: 2.0.0
- node version: 9.10.0  <!-- run `node --version` -->
- npm version: 6.1.0  <!-- run `npm --version` -->
- OS:  Mac<!-- Mac, Linux, Windows -->
```

Others:

**Comment by akires47** [comment]
Hi, some days ago i got a similar error after updating FlexLayoutModule to @latest

<img width="951" alt="screen shot 2018-09-09 at 4 32 26 pm" src="https://user-images.githubusercontent.com/26899212/45268109-3be34900-b44e-11e8-877a-aaded0e2d708.png">

And googling i find the solution, and was updating de typescript version from 2.7.2 to 2.9.2

<img width="202" alt="screen shot 2018-09-09 at 4 32 50 pm" src="https://user-images.githubusercontent.com/26899212/45268111-5c130800-b44e-11e8-8320-9065b1570f74.png">

Hope this helps! 👍

**Comment by sinedied** [comment]
See https://github.com/angular/flex-layout/issues/827

To fix the issue:
- Update TypeScript to `~2.9.2` as @akires47 said
- Update `@angular/flex-layout` to `^6.0.0-beta.18`
- Update `rxjs` to `^6.3.2` 

And you should be good.

It should have been resolved anyway with the latest `5.1.0` generator version, so I'm closing this issue.
